channels/lu.m3u

```diff
@@ -1,12 +1,12 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="ChamberTV.lu" tvg-country="LU" tvg-language="Luxembourgish" tvg-logo="https://i.imgur.com/YldMnap.jpg" group-title="",Chamber TV (1080p)
-https://media02.webtvlive.eu/chd-edge/_definst_/smil:chamber_tv_hd.smil/playlist.m3u8
+https://media02.webtvlive.eu/chd-edge/smil:chamber_tv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="eldoTV.lu" tvg-country="LU" tvg-language="Luxembourgish" tvg-logo="https://i.imgur.com/4ntor8S.png" group-title="",eldo.TV (720p)
 https://eldo-streaming.eldo.lu/eldotv/smil:eldotv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="JasminTV.lu" tvg-country="LU" tvg-language="English" tvg-logo="https://www.parsatv.com/index_files/channels/jasmintv.png" group-title="XXX",Jasmin TV (720p)
-http://109.71.162.112:1935/live/hd.jasminchannel.stream/playlist.m3u8
+https://109.71.162.112/live/hd.jasminchannel.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="JasminTV.lu" tvg-country="LU" tvg-language="English" tvg-logo="https://www.parsatv.com/index_files/channels/jasmintv.png" group-title="XXX",Jasmin TV (720p)
-http://109.71.162.112:1935/live/sd.jasminchannel.stream/playlist.m3u8
+https://109.71.162.112/live/sd.jasminchannel.stream/playlist.m3u8
```

Comment:
Does play but VLC throwing certificate error:

[00007fc56429d3a0] gnutls tls client error: Certificate verification failure: The certificate is NOT trusted. The name in the certificate does not match the expected.